imagej/imagej-ops2/src/main/java/net/imagej/ops2/deconvolve/RichardsonLucyF.java

```diff
@@ -182,22 +184,27 @@ private void computeFilter(final RandomAccessibleInterval<I> input,
 	 *
 	 * @param input
 	 * @param kernel
-	 * @param borderSize
-	 * @param obfInput
-	 * @param obfKernel
 	 * @param outType
-	 * @param fftType
+	 * @param complexType
 	 * @param maxIterations max number of iterations
 	 * @param nonCirculant indicates whether to use non-circulant edge handling
 	 * @param accelerate indicates whether or not to use acceleration
 	 * @param executorService
+	 * @param borderSize (required = false)
+	 * @param obfInput (required = false)
+	 * @param obfKernel (required = false)
 	 * @return the output
 	 */
 	@Override
 	public RandomAccessibleInterval<O> apply(RandomAccessibleInterval<I> input, RandomAccessibleInterval<K> kernel,
-			long[] borderSize, OutOfBoundsFactory<I, RandomAccessibleInterval<I>> obfInput,
-			OutOfBoundsFactory<K, RandomAccessibleInterval<K>> obfKernel, O outType, C complexType,
-			Integer maxIterations, Boolean nonCirculant, Boolean accelerate, ExecutorService es) {
+			O outType, C complexType, Integer maxIterations,
+			Boolean nonCirculant, Boolean accelerate, ExecutorService executorService,
+			@Optional long[] borderSize,
+			@Optional OutOfBoundsFactory<I, RandomAccessibleInterval<I>> obfInput,
+			@Optional OutOfBoundsFactory<K, RandomAccessibleInterval<K>> obfKernel)
```

Comment:
My first pass was to remove it. Then I realized it is probably there intentionally to allow consistent calls for future optimization. Since it's optional anyway I decided to leave it.